src/fhir/input/fsh/profiles/MessageHeader.fsh

```diff
@@ -7,13 +7,7 @@ Description: "MessageHeader des MessageBundles"
 
 * event[x] from ServiceIdentifierVS (required)
 
-* meta.profile 1..* MS
```

Comment:
Es gibt noch ein RuleSet "MetaProfile", sollen wir das irgendwie nutzen?
Kann im Profil die URL als Beispiel eingefügt werden? Es ist zumindest nicht verboten in meta.profile einzutragen, dass die Instanz behauptet konform zu einem Profil zu sein.

Ggf. Frage an geyfyra

Reply:
In den Beispielen muss `meta.profile` nicht explizit gesetzt werden, weil diese Information automatisch auf Grund des zu instanziierenden Profiles gesetzt wird.